### ISSUE TITLE
Add real player models

### ISSUE THREAD

**Issue by Arkanoid0** [momentum-mod/game#518]
Submitting this here as a feature request because I think it is important enough to warrant it and should be done soon, by 1.0.0 at the latest. Our artists have been having much fun working on weapon models, but currently one 1 of 3 modes uses weapon models, while player models are in every game mode, and contribute much more to the feel of the game.

Good player models always become a heavily associated with the game itself, and I feel like much more thought needs to be put into this aspect of the game.

**Comment by Gocnak** [comment]
Thank you for the suggestion. It can be added as fast as modelers and animators get to it, but will definitely be added before 1.0.0.

Also make sure to use [our new templates](https://github.com/momentum-mod/game/issues/new/choose) for things in the future, they help structure things like feature requests and bugs 🙂

**Comment by CoopJax** [comment]
Yeah, I agree. What if you could just use the default hl2 playermodel but you could change the color of it?

**Comment by tsa96** [comment]
Superseded by #1836, which is already in the 0.10.0 board!
